content/applications/sales/sales/ebay_connector/setup.rst

```diff
@@ -1,170 +1,399 @@
 
-#. Activate the :ref:`developer mode <developer-mode>`.
-#. Go to :menuselection:`Settings --> Technical --> Scheduled Actions`
-#. Archive the old synchronization actions (both are named *Ebay: update product status*)
-#. Activate the new synchronization actions (*Ebay: get new orders* which runs every 15min by
-   default and *Ebay: synchronise stock (for 'get new orders' synchronisation)* which runs once a
-   day per default)
-#. Ensure that the **Next Execution Date** for both these actions are in the near future
+The previously copied :guilabel:`User Token` and the :guilabel:`Application Keyset` is now ready to
+be entered into the Odoo database.
+
+Navigate back to Odoo in eBay settings (:menuselection:`Sales app --> Configuration --> Settings -->
+eBay`) and paste the following credentials from eBay into the corresponding fields in Odoo.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Platform
+     - Dev Key/ID
+     - Token
+     - App Key/ID
+     - Cert Key/ID
+   * - eBay
+     - Dev ID
+     - User Token
+     - App ID (Client ID)
+     - Cert ID (Client Secret)
+   * - Odoo
+     - Developer Key
+     - Production/Sandbox Token
+     - Production/Sandbox App Key
+     - Production/Sandbox Cert Key
 
-Starting with the next execution date, the new method will be used instead of the old one.
+.. important::
+   The :guilabel:`Application Keyset` can be accessed by going to `eBay's developer portal
+   <https://go.developer.ebay.com/>`_ and navigate to the `Hi [username]` at top right of screen,
+   then click on :guilabel:`Application Keysets`. Get to the *User Token* in eBay by navigating to
+   the `Hi [username]` at top right of screen, then :guilabel:`User Access Tokens` and click on
+   :guilabel:`Sign in to Sandbox`. The :guilabel:`User Token` can also be accessed by
+   clicking on :guilabel:`User Tokens` from the :menuselection:`Application Keys` page.
+
+Test the setup is correct by saving the credentials in Odoo. Once the initial setup is complete a
+new menu tab in products will appear called `eBay` with the option to :guilabel:`Sell on eBay`.
+See the :doc:`manage` documentation on how to list products.
+
+.. tip::
+   Sync Product Categories by clicking on :guilabel:`Product Categories`. Once the categories are
+   synced a new menu item should appear called `eBay Category` with eBay categories brought over
+   into the database. These are available when listing an eBay item through Odoo.
+
+   .. important::
+      If Product Categories beyond 4 paths are required, users will need to manually add those
+      paths. This has historically been done by getting a list of all product categories beyond 4
+      paths, manually importing them into the Product Category model in Odoo, then linking them
+      individually to the product.
+
+.. seealso::
+   Now that the setup is complete, proceed to either :doc:`create listings <manage>` or :doc:`link
+   existing listings <linking_listings>`.
```

Comment:
what you currently have totally works! I just wanted to suggest another option, which is creating a custom admonition block:
```suggestion
.. admonition:: Read more
   Now that the setup is complete, proceed to either:
   - :doc:`create listings <manage>`
   - :doc:`link existing listings <linking_listings>`.
```